utils/src/main/java/com/cloud/utils/PropertiesUtil.java

```diff
@@ -34,6 +34,10 @@
 public class PropertiesUtil {
     private static final Logger s_logger = Logger.getLogger(PropertiesUtil.class);
 
+    public static String getDefaultApiCommandsFileName() {
+        return "commands.properties";
+    }
```

Comment:
@bhaisaab my thinking is that commands.properties is only managed/used by the Authenticator mechanism.  Is it used more widely?